services/ui-src/src/App.js

```diff
@@ -35,8 +36,36 @@ function App() {
 
     userHasAuthenticated(false);
 
-    history.push("/login");
+    history.push("/");
+  }
+
+  async function handleLogin(event) {
+    event.preventDefault();
+    try {
+      const localLogin = config.LOCAL_LOGIN === "true";
+      if (localLogin) {
+        const alice = {
+          username: "alice",
+          attributes: {
+            given_name: "Alice",
+            family_name: "Foo",
+            email: "alice@example.com",
+          },
+        };
+        loginLocalUser(alice);
+        userHasAuthenticated(true);
+      } else {
+        const authConfig = Auth.configure();
+        const { domain, redirectSignIn, responseType } = authConfig.oauth;
+        const clientId = authConfig.userPoolWebClientId;
+        const url = `https://${domain}/oauth2/authorize?redirect_uri=${redirectSignIn}&response_type=${responseType}&client_id=${clientId}`;
+        window.location.assign(url);
```

Comment:
Here is where your browser is directed to the Cognito hosted ui, where you can login with user/pw or okta